cmd/teleporter-cli/transaction.go

```diff
@@ -0,0 +1,100 @@
+// (c) 2023, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package main
+
+import (
+	"context"
+
+	warpPayload "github.com/ava-labs/avalanchego/vms/platformvm/warp/payload"
+	"github.com/ava-labs/subnet-evm/ethclient"
+	"github.com/ava-labs/subnet-evm/x/warp"
+	teleportermessenger "github.com/ava-labs/teleporter/abi-bindings/go/Teleporter/TeleporterMessenger"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+const (
+	warpPrecompileAddress = "0x0200000000000000000000000000000000000005"
+)
+
+var (
+	rpcEndpoint       string
+	teleporterAddress common.Address
+	client            ethclient.Client
+)
+
+var transactionCmd = &cobra.Command{
+	Use:   "transaction --rpc RPC_URL --teleporter-address CONTRACT_ADDRESS TRANSACTION_HASH",
+	Short: "Parses relevant Teleporter logs from a transaction",
+	Long: `Given a transaction this command looks through the transaction's receipt
+for Teleporter and Warp log events. When corresponding log events are found,
+the command parses to log event fields to a more human readable format.`,
+	Args: cobra.ExactArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		receipt, err := client.TransactionReceipt(context.Background(),
+			common.HexToHash(args[0]))
+		cobra.CheckErr(err)
+		cmd.Print("HELLOOOO")
+		logger.Info("TEST")
```

Comment:
I think you left these here by accident :)

Reply:
haha yes thanks for finding